dashboard/db/migrate/20161111020106_create_pd_enrollment_notifications.rb

```diff
@@ -0,0 +1,11 @@
+class CreatePdEnrollmentNotifications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :pd_enrollment_notifications do |t|
+      t.timestamps
+      t.belongs_to :pd_enrollment, null: false
+
+      # The name of notification: i.e. name of action in Pd::WorkshopMailer
+      t.string :name
```

Comment:
Sounds like you're on board with adding a `comment`, so in that case I think `name` sounds fine. 👍